docs/_pages/contributing.md

```diff
@@ -130,6 +126,65 @@ and more importantly also try to improve the process along the way as well.
    - Before your change can be merged, it should be squashed into a single commit for cleaner commit history.
  - Finally, once your pull request is merged, make sure to `Close` the JIRA.
 
+### Coding guidelines 
+
+Our code can benefit from contributors speaking the same "language" when authoring code. After all, it gets read lot more than it gets
+written. So optimizing for "reads" is a good goal. The list below is a set of guidelines, that contributors strive to upkeep and reflective 
+of how we want to evolve our code in the future.
```

Comment:
`reflective of` -> `reflect on`?

Reply:
I think current phrasing is actually correct